about.html

```diff
@@ -48,7 +52,7 @@ <h3>"Hello! Pets lovers"</h3>
             <h1>Veterinary international meet alternative natural food</h1>
         </div>
         
-        <p>The meeting to present an alternative diet for our pets is held 
+        <p id="introDesc">The meeting to present an alternative diet for our pets is held 
             annually to present the advances in this field. A more balanced 
             and natural diet represents an improvement in the quality of life 
```

Comment:
- [ ] Kindly change the background color to what we have in the design as  raised earlier, our ability to reproduce a given template perfectly shows how well we pay attention to details. 👍 

## Your Design

<img width="525" alt="Untitled" src="https://github.com/Dachrono/capstone/assets/103386183/719d7b11-9a31-4119-8de0-cac3ec8a4ccd">

## Figma Design

<img width="309" alt="Untitled" src="https://github.com/Dachrono/capstone/assets/103386183/de7a8bd4-c4df-486d-9e8a-3cd69d4e1c1a">